packages/governance/src/contractGovernance/typedParamManager.js

```diff
@@ -7,6 +7,14 @@ import { makeParamManagerBuilder } from './paramManager.js';
 /**
  * @typedef {Record<Keyword, ParamType>} ParamTypesMap
  */
+/**
+ * @template {Record<Keyword, ParamRecord>} M
+ * @typedef {{ [R in keyof M]: M[R]['type']}} ParamTypesMapFromRecords
+ */
+/**
+ * @template {ParamTypesMap} M
+ * @typedef {{ [T in keyof M]: { type: M[T], value: ParamValueForType<M[T]> } }} ParamRecordsFromTypes
+ */
```

Comment:
to convert between the actual "record" containing `type` with `value` and the simpler type information.